src/Compilers/VisualBasic/Desktop/CommandLine/CommandLineParser.vb

```diff
@@ -95,9 +95,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim documentationPath As String = Nothing
             Dim errorLogPath As String = Nothing
             Dim parseDocumentationComments As Boolean = False ' Don't just null check documentationFileName because we want to do this even if the file name is invalid.
-            Dim outputKind As OutputKind = OutputKind.ConsoleApplication
+            Dim outputKind As outputKind = outputKind.ConsoleApplication
             Dim ssVersion As SubsystemVersion = SubsystemVersion.None
-            Dim languageVersion As LanguageVersion = LanguageVersion.VisualBasic14
+            Dim languageVersion As languageVersion = languageVersion.VisualBasic14
             Dim mainTypeName As String = Nothing
```

Comment:
@VSadov I did not make this changes hence it must have been from an automatic reformat.  I will update.

CC @basoundr